query-engine/prisma/src/error.rs

```diff
@@ -57,6 +57,7 @@ impl PrismaError {
 
                     error.pretty_print(&mut message, file_name, dml_string)?
                 }
+                write!(message, "Validation Error Count: {}", errors.to_iter().len())?;
```

Comment:
That was easy :p

Reply:
It's not hard to turn the screw, it's hard to know which screw to turn ;-)

Reply:
So beautiful!